backend/app/db/models/litiguation.py

```diff
@@ -0,0 +1,187 @@
+import enum
+from sqlalchemy import (
+    Column,
+    Enum,
+    SmallInteger,
+    Integer,
+    Text,
+    ForeignKey,
+    Date,
+)
+
+from app.db.session import Base
+from . import Case, Keyword, Document
+
+
+class LitCaseKeyword(Base):  # noqa: D101
+    __tablename__ = "lit_case_keyword"
+
+    keyword_id = Column(
+        Integer, ForeignKey(Keyword.id), nullable=False, primary_key=True
+    )
+    case_id = Column(
+        Integer,
+        ForeignKey(Case.id, ondelete="CASCADE"),
+        nullable=False,
+        primary_key=True,
+    )
+
+
+# --------------------------------------------------
+# Laws (Internal & External)
```

Comment:
@joel-wright I think this is where we should have the discussion about if we have a single type of "ReferencedLaw" object that can be either internal / external with all the fields on we need. At the end of the day we just need to record the referenced law and whether its internal or external is something that can be determined (or change) later.

Reply:
That is true - it might incur a higher maintainance when updating than the saving it has when creating - let me scratch that up to a bad idea :smile: